Scripts/SetupFunctionalTests.ps1

```diff
@@ -1,78 +1,73 @@
-param (
-    [string]$NuGetRoot=$null
-)
+. "$PSScriptRoot\Utils.ps1"
 
 function EnableWindowsDeveloperMode()
 {
     $windowsDeveloperModeKey = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"
-    if (Test-Path $windowsDeveloperModeKey)
+    $success = SetRegistryKey $windowsDeveloperModeKey "AllowDevelopmentWithoutDevLicense" 1 -NeedAdmin
+    if (!($success))
     {
-        Set-ItemProperty -Path $windowsDeveloperModeKey -Name AllowDevelopmentWithoutDevLicense -Value 1
-        return $true
-    }
-    else {
-        return $false
+        exit 1
```

Comment:
From maintenance perspective it'd be better not to exit directly from functions. It's hard to troubleshoot the script behavior. As an alternative if that's a fatal error you can throw an exception with actionable message.